src/CircleEditor/Editor.ts

```diff
@@ -0,0 +1,304 @@
+/*
+ * Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {Balloon} from '../Utils/Balloon';
+import * as Types from './CircleType';
+
+import * as flexbuffers from 'flatbuffers/js/flexbuffers';
+
+import * as Circle from './circle_schema_generated';
+
+export interface Editor {
+  edit(targetName: string, targetData: any): void;
+}
+
+export class AttributeEditor {
+  private _model: Circle.ModelT;
+  private _subgraphIdx: number = -1;
+  private _operatorIdx: number = -1;
+
+  public constructor(model: Circle.ModelT) {
+    this._model = model;
+  }
+
+  private guessExactType(n: any) {
+    if (isNaN(Number(n))) {
+      return 'error';
+    }
+
+    if (Number(n) % 1 === 0) {
+      return 'int';
+    } else if (Number(n) % 1 !== 0) {
+      return 'float';
+    } else {
+      return 'not supported type';
+    }
+  }
+
+  public edit(inputBuiltinCode: string, attr: any): void {
+    if (inputBuiltinCode === undefined || this._subgraphIdx === -1 || this._operatorIdx === -1) {
+      Balloon.error('input data is undefined', false);
+      return;
+    }
+    const operator: any = this._model.subgraphs[this._subgraphIdx].operators[this._operatorIdx];
```

Comment:
I am not sure but `subgraphIdx` maybe larger than the length of `subgraphs`.
How about adding a guard for it or apply following? :)

```suggestion
    const operator: any = this._model.subgraphs[this._subgraphIdx]?.operators[this._operatorIdx];
```